ghe-check-hotpatch.sh

```diff
@@ -55,7 +59,11 @@ sanity_check () {
 # Verify that the same major version is reported to help rule out typos or
 # attempting to run against feature release upgrades.
 
-  API_VERSION=$(curl -s http://localhost:1337/api/v3/meta | jq .installed_version | tr '"' ' ' | xargs)
+  API_VERSION="$(curl -s http://localhost:1337/api/v3/meta | jq -r .installed_version)" || {
+    echo "ERROR: Failed to retrieve version via API"
+    export FAILURE_STATUS=true
+    return
+  }
```

Comment:
`tr '"' ' '` introduces spaces that `xargs` can remove, however `tr -d '"'` could remove them without that need. Regardless, `jq`s `-r` option outputs the value raw, negating the need to manipulate it further (and would allow for the output to, for example, contain `"`)